src/stdio.jl

```diff
@@ -23,16 +23,6 @@ Base.displaysize(io::IJuliaStdio) = displaysize(io.io)
 Base.unwrapcontext(io::IJuliaStdio) = Base.unwrapcontext(io.io)
 Base.setup_stdio(io::IJuliaStdio, readable::Bool) = Base.setup_stdio(io.io.io, readable)
 
-for s in ("stdout", "stderr", "stdin")
```

Comment:
You should put this in a conditional to still support older Julia version:
```
if VERSION < v"1.7.0-DEV.254"
    # ...
end
```

Reply:
Ah, of course. Done, testing now...